include/aws/s3/private/s3_auto_ranged_put.h

```diff
@@ -39,8 +41,12 @@ struct aws_s3_auto_ranged_put {
 
     /* Members to only be used when the mutex in the base type is locked. */
     struct {
+        /* Array list of `struct aws_string *` */
         struct aws_array_list etag_list;
 
+        struct aws_s3_paginated_operation *list_parts_operation;
+        struct aws_string *continuation_token;
```

Comment:
renaming it to `list_parts_continuation_token`? It's for list parts right? It's confusing with the put object is also paused/resume(continues)